tests/foreman/cli/test_host_collection.py

```diff
@@ -28,258 +28,247 @@
 from robottelo.constants import DEFAULT_CV
 from robottelo.constants import ENVIRONMENT
 from robottelo.datafactory import invalid_values_list
+from robottelo.datafactory import parametrized
 from robottelo.datafactory import valid_data_list
-from robottelo.test import CLITestCase
 
 
-class HostCollectionTestCase(CLITestCase):
-    """Tests for Host Collections via Hammer CLI"""
-
-    @classmethod
-    def setUpClass(cls):
-        """Prepare some data to be used in tests"""
-        super().setUpClass()
-        cls.organization = make_org()
-        cls.library = LifecycleEnvironment.info(
-            {'organization-id': cls.organization['id'], 'name': ENVIRONMENT}
+@pytest.fixture(scope='module', autouse=True)
+def set_up_a_host_collection(module_org):
+    """ Make a host collection for use in tests"""
+    make_host_collection({'organization-id': module_org.id})
```

Comment:
Why is this fixture necessary for every test function? From what I can tell, every test is calling `make_host_collection` itself, and no tests are actually using or dependent on the host collection created here?

Reply:
I see only the end to end test is using two hosts collections. I think there is no need for two, I will remove it.